src/Controls/src/Core/ViewExtensions.cs

```diff
@@ -519,5 +519,24 @@ static internal void MapFocus(this VisualElement view, FocusRequest focusRequest
 
 			return window.Handler?.MauiContext;
 		}
+
+		/// <summary>
+		/// TBD
+		/// </summary>
+		/// <param name="view"></param>
+#pragma warning disable RS0016 // Add public types and members to the declared API
```

Comment:
Do we want to add this API for net9 ?

Reply:
Yea, I'm waiting for confirmation from the team if we want to add these :-)